app/main.py

```diff
@@ -1 +1,39 @@
-#  write your code here
+class Animal:
+    animals_alive = []
+
+    def __init__(self, name: "str", appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.animals_alive.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
This is a good example of where you can avoid the use of an `else` statement. Since you return from the function in the `if` branch, you can just write `return 0` after the `if` statement, without the `else`. This will make your code cleaner and easier to read. (Checklist: Make Code Easier, point 1)